Sources/Commands/SwiftPackageTool.swift

```diff
@@ -594,7 +594,9 @@ extension SwiftPackageTool {
                 skipSynthesizedMembers: skipSynthesizedMembers,
                 minimumAccessLevel: minimumAccessLevel,
                 skipInheritedDocs: skipInheritedDocs,
-                includeSPISymbols: includeSPISymbols)
+                includeSPISymbols: includeSPISymbols,
+                outputFormat: SymbolGraphExtract.OutputFormat.json(pretty: prettyPrint)
```

Comment:
nit: `outputFormat: .json(pretty: prettyPrint)`

Reply:
somewhat unrelated: is JSON the only output we want to support, or should this be extended to allow users to also specify the output format?

Reply:
Thanks, resolved in https://github.com/apple/swift-package-manager/pull/5495/commits/b0a04988fa4cedcefd28392f2be53f4132b55fe8

Reply:
> somewhat unrelated: is JSON the only output we want to support, or should this be extended to allow users to also specify the output format?

The symbol graphs, even for smaller packages, are verbose and get quite dense with data quickly so if you're thinking of other formats that are human readable I'm not sure if say yaml or so would actually add value to the human reading the data. In any case, reading the symbol graph in an editor is rather an edge case.

I think it could be interesting if there was another format that would help speed/performance. For example DocC has a "mode" where it uses a custom concurrent JSON decoder https://github.com/apple/swift-docc/blob/main/Sources/SwiftDocC/Infrastructure/Symbol%20Graph/SymbolGraphConcurrentDecoder.swift#L45-L48 to concurrently decode a symbol graph to avoid slowing down the build process while it needs to decode the graph before building the documentation. 

It could be interesting if the compiler did output bitstream so that symbol information can be decoded on-the-fly while the compiler generates it (or even binary data that another process can read in real time) but in the end this would involve changes in the compiler https://github.com/apple/swift/blob/1b5c3d79dd8eebe1f5c6c88b715083951c3de806/lib/SymbolGraphGen/SymbolGraph.cpp#L471 which would maybe feel like extra work without a concrete and immediate use-case.